pyproject.toml

```diff
@@ -3,15 +3,15 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 [project]
 name = "one_py_sdk"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
Is this a bugfix change? If it's adding functionality, I think SemVer says this should become 1.1.0.